contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -253,4 +256,34 @@ public final void testWritePrivateProfileSection() throws IOException {
             reader.close();
         }        
     }
+    
+    public void testGetResource() {
+		// On Windows 7, "14" is the type assigned to the "My Computer" icon
+		// (which is named "ICO_MYCOMPUTER")
+    	byte[] results = Kernel32Util.getResource("c:\\windows\\explorer.exe", "14", "ICO_MYCOMPUTER");
```

Comment:
Obviously, nothing that harms or changes the computer on which the tests are running - e.g., I would **not** test _FormatDisk / Shutdown / Logoff / Reboot_ API calls, modify sensitive registry values, modify the permanent environment, remove the host from its Windows domain, change the display settings,  corrupt .exe files, etc., etc., etc.... Basically, all tests should be as harmless as possible - especially if they fail. We would not want a failing test to leave the host that is running it in a corrupted state - at the very least a reboot should **always** fix any invalid state that was created due to a failing test. Any such situation this should be (extremely) **well documented**.